styles.css

```diff
@@ -8,14 +8,19 @@ body {
   font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
 /* header */
 
 header .container {
   margin: 30px auto;
   width: 40%;
   display: flex;
   justify-content: space-between;
-  /* position: absolute; */
+  position: absolute;
+  left: 29%;
```

Comment:
this is a code smell - a code smell means there is not something inherently wrong but it is unusual to see % units on position values so i suspect there is a better or easier way of doing what you want.